concourse/scripts/deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+SCRIPT_DIR=$(cd $(dirname $0) && pwd)
+
+env=${DEPLOY_ENV-$1}
+pipeline="cf-deploy"
+config="${SCRIPT_DIR}/../pipelines/deploy.yml"
+
+[[ -z "${env}" ]] && echo "Must provide environment name" && exit 100
+bash "${SCRIPT_DIR}/deploy-pipeline.sh" "${env}" "${pipeline}" "${config}"
+
+
+export ATC_URL=${ATC_URL:-"http://192.168.100.4:8080"}
+export fly_target=${FLY_TARGET:-tutorial}
+
+fly unpause-pipeline --pipeline "${pipeline}"
+curl "${ATC_URL}/pipelines/${pipeline}/jobs/init-bucket/builds" -X POST
```

Comment:
When you do the curl, you get the build number in the response in a `Location` header:

```
$ curl -I http://192.168.100.4:8080/pipelines/cf-deploy/jobs/init-bucket/builds -X POST
HTTP/1.1 302 Found
Location: /pipelines/cf-deploy/jobs/init-bucket/builds/15
Date: Mon, 07 Dec 2015 10:57:57 GMT
Content-Length: 0
Content-Type: text/plain; charset=utf-8
```

The fly watch command has a `-b` option to specify the build, but I am not sure how that works:

```
$ fly watch -h
error: Usage:
  fly [OPTIONS] watch [watch-OPTIONS]

Application Options:
  -t, --target=                 Concourse target name or URL (default: http://192.168.100.4:8080)

Help Options:
  -h, --help                    Show this help message

[watch command options]
      -j, --job=PIPELINE/JOB    Watches builds of the given job
      -b, --build=              Watches a specific build

```